1주차 과제.css

```
@@ -0,0 +1,159 @@
+body{
+    margin:0;
+    padding:0;
+}
+.banner{
+    background-color: #4B89DC;
+    width:100%;
+    height:64px;
+    border: none;
+    margin-top:0px;
+    
+}
```

```suggestion
body{
    margin:0;
    padding:0;
}

.banner{
    background-color: #4B89DC;
    width:100%;
    height:64px;
    border: none;
    margin-top:0px;   
}
```

팀 컨벤션에 따라 혹은 개인의 성향에 따라 다르겠지만 보통 css는 가독성 측면을 위해 각 요소마다 개행을 넣어주는 편이에요